pkg/kubelet/cm/cpumanager/policy_test.go

```diff
@@ -540,6 +515,108 @@ var (
 			79: {CoreID: 39, SocketID: 1, NUMANodeID: 3},
 		},
 	}
+	/*
+		Topology type 2 from dual xeon gold 6230; lscpu excerpt
```

Comment:
"topology with numa sub clustering"?

Reply:
Base the information I have searched(maybe I am wrong), both this new Topology and Topology of Gold 6230 now
 in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/policy_test.go#L418 have SNC enabled. 
If it's hard to name this, I think we could load similar silicon to 6230, such as Gold 6226(https://www.spec.org/cpu2017/results/res2019q3/cpu2017-20190722-16283.html).

What do you think? @ffromani

Reply:
looks good enough, no need to change then